### ISSUE TITLE
igluctl: ensure '-' -> '_' when generating jsonpath files

### ISSUE THREAD

**Issue by yalisassoon** [snowplow/iglu#226]
Currently there's a bug where if there's a schema name e.g.

```
com.mycompany/ad-view/jsonschema/1-0-0
```

Then running 

```
$ /path/to/igluctl ./schemas/com.mycompany/ad-view
```

Generates a jsonpath file name `ad-view_1.json` instead of `ad_view_1.json`.

**Comment by alexanderdean** [comment]
Ouch - nasty bug. @chuwy please make sure you also cross-check the rest of the rules in `make_sql_safe`:

https://github.com/snowplow/snowplow/blob/master/4-storage/storage-loader/lib/snowplow-storage-loader/shredded_type.rb#L172

**Comment by oguzhanunlu** [comment]
This is fixed through #271.
